README.md

```diff
@@ -1,123 +1,91 @@
 
-### OS X
+# Download jujubox
 
-Docker does not run natively in OS X, so we'll need to use VirtualBox
-and boot2docker. This is relatively easy for users of brew:
-
-```
-brew install cask
-brew cask install virtualbox
-brew install docker
-brew install boot2docker
-
-boot2docker init
-boot2docker up
-```
-
-When boot2docker finishes, it will prompt you to export Docker
-environment variables. With that done, you're ready to use docker.
-You do not need to run the docker commands using sudo because
-boot2docker runs as root.
-
-    docker run  -ti jujusolutions/jujubox
-
-# Download
-
-Download the Juju Box source from github or the container from
-[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/):
-
-### Linux
-You can clone the repository from github.
+If you want to see the code you can download the jujubox project from 
+[github](https://github.com/juju-solutions/jujubox) and build it from source.
 
 ```
 git clone https://github.com/jujusolutions/jujubox.git && cd jujubox
-
+docker build -t jujusolutions/jujubox ./
 ```
 
-Or download and install the container directly from hub.docker.io:
+Or simply pull the jujubox container from
+[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/).
 
 ```
-sudo docker run  -ti jujusolutions/jujubox
+docker pull jujusolutions/jujubox
 ```
 
+# Run jujubox
 
-# To Run
+There are several different ways to run jujubox depending on what you are 
+trying to accomplish. 
 
-There are two ways to run the docker container, from the github
-repository (build from source) or from the container directly from the
-hub.docker.io.
+## Running jujubox to evaluate Juju
 
+If you do not already have Juju installed and want to check this container out 
+you can simply run the container in an interactive (-i) tty (-t) session to use
+the Juju tools:
 
-### Linux
-
-#### From source:
 ```
-sudo docker build -t jujubox ./
-sudo docker run -i -t jujubox:latest
+docker run --rm -it jujusolutions/jujubox 
 ```
 
-#### Use hub.docker.io:
-```
-sudo docker run  -ti jujusolutions/jujubox
-```
+## Running jujubox if you have Juju installed on host
 
-If you already have juju installed, run the container using your
-existing ${JUJU_HOME} directory (~/.juju):
-
-```
-sudo docker run --rm -v ${JUJU_HOME}:/home/ubuntu/.juju -it jujusolutions/jujubox
-```
+If you already have Juju installed on the host machine, you can mount the 
+existing host directories to the container using volume mounts (-v). Run the 
+container using your volume mounts to your existing ${JUJU_HOME} directory 
+(~/.juju) and your existing ${JUJU_REPOSITORY} directory (where the charms
+are held on the host):
 
-### OS X
-Remember that boot2docker runs as root, so don't use `sudo` for the docker
-commands:
 ```
-docker run -ti jujusolutions/jujubox
+docker run --rm -it \
+  -v ~/.local/share/juju:/home/ubuntu/.local/share/juju \
+  -v ${JUJU_REPOSITORY}:/home/ubuntu/ \
```

Comment:
Correct and good catch. I remember having this discussion with @johnsca in Colorado about having a "charms" directory or not. I am afraid to change this as it could break existing tools or use cases. We should discuss this change here. @chuckbutler can you weigh in on this issue too?